packages/terra-date-picker/src/terra-dev-site/doc/example/DatePickerControlled.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import Field from 'terra-form-field';
+// eslint-disable-next-line import/no-unresolved, import/extensions, import/no-extraneous-dependencies
+import DatePicker from 'terra-date-picker/lib/DatePicker';
+
+const propTypes = {
+  /**
+   * The date value used for a controlled component.
+   */
+  value: PropTypes.string,
+};
+
+const defaultProps = {
+  value: undefined,
+};
+
+class DatePickerExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { date: this.props.value };
+    this.handleDateChange = this.handleDateChange.bind(this);
+    this.handleDateChangeRaw = this.handleDateChangeRaw.bind(this);
+  }
+
+  handleDateChange(event, date) {
+    this.setState({ date });
+  }
+
+  handleDateChangeRaw(event, date) {
+    this.setState({ date });
+  }
+
+  render() {
+    return (
+      <div>
+        <p>
+Selected ISO Date:
+          <span style={{ display: 'inline-block' }}>{this.state.date}</span>
```

Comment:
Need a class for the inline style here

Reply:
Good call. https://github.com/cerner/terra-framework/pull/738/commits/d3647e3bcc31b6316300d192059c267647cbd3c3